packages/docs-app/src/styles/_examples.scss

```diff
@@ -189,6 +189,18 @@
   max-width: 280px;
 }
 
+#{example("OverflowListExample")} {
+  .#{$ns}-card {
+    margin: 0;
+    min-width: $pt-grid-size * 7;
+  }
+
+  .#{$ns}-overflow-list {
+    flex-wrap: nowrap;
+    white-space: nowrap;
```

Comment:
@invliD 🤔 why are these styles not on the component itself? seems fairly important actually.

Reply:
I don't think they should be there. `flex-wrap: nowrap` is actually the default, so I'll remove it. `white-space: nowrap` is necessary in this specific case, because I don't want the each breadcrumb to wrap within its container, but I don't think this is a generalizable assumption.